app/src/main/AndroidManifest.xml

```diff
@@ -71,6 +73,7 @@
 
         <activity android:name=".activity.MainActivity"
             android:launchMode="singleTask"
+            android:exported="true"
```

Comment:
Why is this exported?
Seems like an important change that should be better documented - in the ticket and in commit message.

Reply:
If the app component includes the LAUNCHER category, set android:exported to true.
https://developer.android.com/about/versions/12/behavior-changes-12

Reply:
I don't think `MainActivity` should be exported.

In Fenix, the equivalent is `HomeActivity` and it doesn't seem to need it: https://github.com/mozilla-mobile/fenix/blob/main/app/src/main/AndroidManifest.xml#L44-L56

If you look directly at the activity, instead of the alias too, it is not an exported activity. 

From the docs:
> If the app component includes the LAUNCHER category, set android:exported to true. **In most other cases, set android:exported to false.**

The idea is that we only have one entry point into the app that we can control the flow of intents from i.e. the IntentReceiverActivity, which will then tell the appropriate activity to start up.

Reply:
@ionutbedregeanu ^

Reply:
If we are not setting a value for `android:exported` for `MainActivity` the app doesn't compile and if `android:exported` is set to false we can't launch the app
I'm not sure why in Fenix is working, an investigation should be done